app.js

```diff
@@ -1,9 +1,97 @@
 // in server.js
 const express = require('express');
 const app = express();
+const cors = require('cors')
+const pgp = require('pg-promise')
+const bodyParser = require('body-parser')
+const questions = require('./src/database/queries/questions.js')
 
-app.use(express.static(__dirname + '/public/dist/'));
-app.use(express.static(__dirname + '/src/browser/'));
+app.use( bodyParser.json() )
+app.use( bodyParser.urlencoded( {extended: false}) )
+app.use( express.static(__dirname + '/public/dist/') );
+app.use( express.static(__dirname + '/src/browser/') );
+app.use( ( cors() )
+
+app.get('/users/:name', (request, response) => {
+  const { name } = request.params
+  users.findbyName(name)
+  .then( results => response.json( results ) )
+  .catch( err => console.log('err', err) )
+})
+
+app.get('/users/:github_handle', (request, response) => {
+  const { github_handle } = request.params
+  users.findbyGithub(github_handle)
+  .then( results => response.json( results ) )
+  .catch( err => console.log('err', err) )
+})
+
+app.get('/questions/:tags', (request, response) => {
```

Comment:
also we want a route - `/api/questions` which accepts the  GET request method, and takes 2 query params - `topic` & `difficulty`.